common/api/ecschema-metadata.api.md

```diff
@@ -1662,42 +1674,53 @@ export class Schema implements CustomAttributeContainerProps {
     get writeVersion(): number;
 }
 
-// @beta (undocumented)
+// @alpha (undocumented)
 export class SchemaCache implements ISchemaLocater {
     constructor();
-    addSchema<T extends Schema>(schema: T): Promise<void>;
-    addSchemaSync<T extends Schema>(schema: T): void;
+    addSchema<T extends Schema>(schema: T, loadSchema?: DelayedPromise<T>): Promise<void>;
+    addSchemaSync<T extends Schema>(schema: T, loadSchema?: DelayedPromise<T>): void;
```

Comment:
DelayedPromise in a sync method?

Reply:
The `DelayedPromise` is removed from the sync method's parameters. The sync method's behaviour is unchanged, and it still requires that the schema be fully loaded before the method is called.